### PR TITLE
Prepare for 2020.01.0

### PR DESCRIPTION
**Pull request by datakurre**
* [x] conda 4.8.1
* [x] jupyter-starters 0.2.2a0
* [x] jupytext 1.3.1
* [x] matplotlib 3.1.2
* [x] rise 5.6.0
* [x] robotkernel 1.3

### CODE REVIEW

**Comment by datakurre** [comment]
Green for RF 3.2a1! So cool!

Let’s keep this open for new RF releases (but rebase after 2019.12.0).

**Comment by bollwyvl** [comment]
in the process of cutting 0.2.2a0 of starters...

**Comment by datakurre** [comment]
I will do robotkernel 1.3rc1 with packaged starters. The repository will still include some notebook not in starters, so will still need to include them. I'll try to make the changes here once done.

**Comment by datakurre** [comment]
@bollwyvl Is matplotlib heavy? For some reason, I decided to include it in the quickstart notebook of robotkernel :/

**Comment by datakurre** [comment]
And this was fun discovery. Widgets have been working on RobotKernel for some time (this didn't work last year yet):

rf-notebook-widgets

**Comment by datakurre** [comment]
I decided to keep "examples"-folder so, only tutorial needs to be removed from var/www.

But that seems to be a lot of work. So, docs/notebooks will still exist, but it only contains symlinks into the actual src package where the real notebooks are for starter packaging. We'll see how robotlab build survives it.

**Comment by datakurre** [comment]
starters 0.2.2a0 were looking good at https://mybinder.org/v2/gh/robots-from-jupyter/robotkernel/master?urlpath=lab

**Comment by bollwyvl** [comment]
Yay!

Matplotlib is great, +1 to including.

**Comment by bollwyvl** [comment]
This works:

https://mybinder.org/v2/gh/robots-from-jupyter/robotkernel/master?urlpath=lab/tree/starter/robotkernel-quickstart

**Comment by datakurre** [comment]
It looks like I will need some help to make this release pass the build...

**Comment by bollwyvl** [comment]
You need any support?

**Comment by bollwyvl** [comment]
> but it only contains symlinks into the actual src package where the real notebooks are for starter packaging

Not going to work on windows...

**Comment by bollwyvl** [comment]
:crossed_fingers:

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 BSD 3-Clause License
 
-Copyright (c) 2018, Robots from Jupyter
+Copyright (c) 2020, Robots from Jupyter
```
Comment (bollwyvl) [inline LICENSE:3]: :tada:

File: tests/resources/Selectors.robot [hunk 2]
```diff
@@ -1,10 +1,10 @@
 *** Variables ***
-${XP LAUNCH SECTION}    xpath://h2[contains(@class, 'jp-Launcher-sectionTitle')][contains(text(), 'Starters')]
+${XP LAUNCH SECTION}    xpath://h2[contains(@class, 'jp-Launcher-sectionTitle')][text() = 'Starters']
 ${CSS LAUNCH CARD}    css:[data-category\="Starters"]
-${CSS LAUNCH CARD TUTORIAL}    ${CSS LAUNCH CARD}\[title\="A guided tutorial through using robotkernel"]
+${CSS LAUNCH CARD TUTORIAL}    ${CSS LAUNCH CARD}\[title\="Tutorial for Robot Framework on Jupyter"]
 ${XP FILE TREE ITEM}    xpath://span[contains(@class, 'jp-DirListing-itemText')]
-${XP FILE TREE TUTORIAL}    ${XP FILE TREE ITEM}\[contains(text(), 'robotkernel-tutorial')]
-${XP FILE TREE TUTORIAL 00}    ${XP FILE TREE ITEM}\[contains(text(), '00 JupyterLab User Interface.ipynb')]
+${XP FILE TREE TUTORIAL}    ${XP FILE TREE ITEM}\[text() = 'tutorial']
```
Comment (bollwyvl) [inline tests/resources/Selectors.robot:6]: this was the only thing you had missed for the most recent ci fail... but you know me, can't leave well enough alone...

**Comment by bollwyvl** [comment]
ick. windows 38 vs notebook still broken.

**Comment by datakurre** [comment]
If I understood properly, jupyter-starters's test failed on Windows: https://dev.azure.com/robots-from-jupyter/1e5f1f30-0c52-4d28-8cf7-8c72252fbb49/_apis/build/builds/293/logs/81

**Comment by datakurre** [comment]
Thanks for the support!

I tried to fix that symlink issue by using the original directory from the package.

**Comment by datakurre** [comment]
Thanks a lot again! If this passes, I'll test the installers during the week, fix possible typos on docs, and make a release during the weekend. Just in time for the workshop next week!

**Comment by datakurre** [comment]
:rocket:

**Comment by bollwyvl** [comment]
Huzzah!

**Comment by datakurre** [comment]
Windows looked good. Will do draft release and final testing once robotkernel is ready.

**Comment by bollwyvl** [comment]
A new version of FF 68LTS got into conda-forge last night... our pin should pull it in.

**Comment by datakurre** [comment]
Oh dear. After tagging, MacOSX tests failed and release deploy did not happen :(

https://dev.azure.com/robots-from-jupyter/robots-from-jupyter/_build/results?buildId=304&view=results

**Comment by datakurre** [comment]
Nice, I can delete tag and push it again. https://dev.azure.com/robots-from-jupyter/robots-from-jupyter/_build/results?buildId=306&view=results Let's see...

**Comment by datakurre** [comment]
Tested:

* [x] MacOS
* [ ] Windows
* [ ] Linux

**Comment by datakurre** [comment]
@bollwyvl How have you made the package listings so far? Just for me to know that I get the listing with same method (unless installing and getting from the installation is the only method).

**Comment by datakurre** [comment]
I believe that I found the part from Azure logs that installs RobotLab and lists all the packages. Changelog updated.

**Comment by datakurre** [comment]
@bollwyvl Help needed. It might be just because of my NixOS, but I could not confirm if OpenCV works with Linux installer.

**Pull request merged**